core/filters/test/unit/html-decode.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { htmlDecode } from '@vue-storefront/core/filters/html-decode'
+
+describe('html-decode', () => {
+  it('Check if string is decoded', () => {
+    expect(htmlDecode('foo &copy;')).toBe('foo ©')
+  })
+
+  it('Check if string is empty', () => {
+    expect(htmlDecode()).toBe('')
```

Comment:
`htmlDecode` requires 1 argument. Use some falsy value to test this ternary operator inside